### DISCUSSION TITLE
XObjectImageMask doesn't require ImageMask to be true and uses ColorSpace and Mask in predicates

### DISCUSSION THREAD

**Discussion by bdoubrov** [pdf-association/arlington-pdf-model#53] [General]
As far as I understand, **XObjectImageMask** corresponds to the case when Image XObject is used as an Image Mask. Thus, I would expect that it has only one permitted value (_true_) for **ImageMask** key. 

Both **Mask** and **ColorSpace** are not permitted in Image masks, which is correctly reflected in the **XObjectImageMask** type. However, the restriction in **ImageMask** still uses these keys in the predicates, even if they are no longer present in **XObjectImageMask**.

Is there a special reason for this?

**Comment by faceless2** [comment]
ImageMasks also mentioned in https://github.com/pdf-association/arlington-pdf-model/issues/38 - the model has moved on a bit since then, so mainly cross-linking for context and reference.
